packages/apps/shopify-app-remix/src/server/index.ts

```diff
@@ -10,6 +10,7 @@ export {
   LogSeverity,
   DeliveryMethod,
   BillingInterval,
+  BillingReplacementBehavior,
```

Comment:
We didn't export this before, so it wasn't easy to access it from the remix template when needed.